src/gui/dialogs/campaign_difficulty.cpp

```diff
@@ -77,35 +76,6 @@ config generate_difficulty_config(const config& source)
 	// Populate local config with difficulty children
 	result.append_children(source, "difficulty");
 
-	// Convert legacy format to new-style config if latter not present
-	if(result.empty() && source.has_attribute("difficulties")) {
-		deprecated_message("[campaign]difficulties", DEP_LEVEL::FOR_REMOVAL, {1, 15, 0}, "Use [difficulty] instead.");
```

Comment:
Surely the deprecated_message function call should be retained, and upgraded to DEP_LEVEL::REMOVED?